File: C_Sharp/InterfacesLab/InterfacesLab/MediocreLawnCareProvider.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InterfacesLab
{
    class MediocreLawnCareProvider : ILawnCareProvider
    {
        public bool IsAvailable => true;

        public int Rating => 3;

        public void BookMowing()
        {
        }

        public void BookRaking()
        {
        }

        public void BookWeedRemoval()
        {
        }
    }
}

File: Java/InterfaceLab/src/com/sonicscholar/MediocreLawnCareProvider.java
package com.sonicscholar;

public class MediocreLawnCareProvider implements ILawnCareProvider{

    @Override
    public void bookMowing() {

    }

    @Override
    public void bookRaking() {

    }

    @Override
    public void bookWeedRemoval() {

    }

    @Override
    public boolean isAvailable() {
        return true;
    }

    @Override
    public int getRating() {
        return 3;
    }
}
